src/scenes/Projects/Overview/ProjectOverview.graphql

```diff
@@ -0,0 +1,58 @@
+query ProjectOverview($input: ID!) {
+  project(id: $input) {
+    id
+    name {
+      value
+    }
+    location {
+      value {
+        ...DisplayPlace
+        region {
+          value {
+            name {
+              value
+            }
+          }
+        }
+      }
+    }
+    mouStart {
+      value
+    }
+    mouEnd {
+      value
+    }
+    status
+    budget {
+      value {
+        ...BudgetOverview
+      }
+    }
+    ... on TranslationProject {
```

Comment:
You should be able to define these props without specifying which type (`... on X`) since they exist on the `Project` interface as well

Reply:
Oh :facepalm: I screwed up that interface on API. Just pushed a fix.

Reply:
The fragments are fixed to use the correct interface.